dotCMS/src/main/java/com/dotcms/variant/VariantAPIImpl.java

```diff
@@ -126,4 +137,50 @@ public Optional<Variant> get(final String name) throws DotDataException {
     public List<Variant> getVariants() throws DotDataException {
         return variantFactory.getVariants();
     }
+
+    @Override
+    public void promote(final Variant variant, final User user) throws DotDataException {
```

Comment:
Wrap in trax

Reply:
done https://github.com/dotCMS/core/pull/24450/commits/2359d346af0603b441d48bd3e24772a404b4a906#diff-6b9c195cdec7e7bc927ed858f326518137ab896c29e1d05108f72898d006baf7R141